src/core/control/PageBackgroundChangeController.h

```diff
@@ -38,7 +38,7 @@ class PageBackgroundChangeController:
     virtual void changeCurrentPageBackground(PageType& pageType);
     void changeCurrentPageBackground(PageTypeInfo* info) override;
     void changeAllPagesBackground(const PageType& pt);
-    void insertNewPage(size_t position);
+    void insertNewPage(size_t position, bool shouldScrollToPage = false);
```

Comment:
It looks inconsistent to set the different default values here and in `Control.h`.
```suggestion
    void insertNewPage(size_t position, bool shouldScrollToPage = true);
```